practice/src/javascripts/products/productList.js

```diff
@@ -0,0 +1,19 @@
+import ProductService from '../services/productService';
+import ProductTemplate from '../templates/productTemplate';
+
+export default class ProductList {
+  constructor() {
+    this.productService = new ProductService();
+    this.productContainer = document.querySelector('.homepage-main');
+    this.productTemplate = new ProductTemplate();
```

Comment:
```
const service = new ProductService();
  const template = new ProductTemplate();
  const productList = new ProductList(service, template);
```

I have new ProductService and ProductTemplate above, why do you need to new it here again?